cesium_app/ext/sklearn_models.py

```diff
@@ -172,7 +179,16 @@ def verify_type(value, vtype):
 
         try:
             verify_type(param_value, required_type)
+            if (make_list(param_value) == param_value and list not in
```

Comment:
Some similar logic, but different enough that they couldn't both use the same utility function